base/shell.jl

```diff
@@ -344,7 +344,7 @@ function shell_escape_csh(io::IO, args::AbstractString...)
 end
 shell_escape_csh(args::AbstractString...) =
     sprint(shell_escape_csh, args...;
-           sizehint = sum(sizeof.(args)) + length(args) * 3)
+           sizehint = mapreduce(sizeof, +, args; init = 0) + length(args) * 3)
```

Comment:
```suggestion
           sizehint = sum(sizeof, args) + length(args) * 3)
```

This is probably a bit more readable, and infers just as well (much better than the broadcast)